pom.xml

```diff
@@ -139,7 +139,6 @@
             <groupId>javax.transaction</groupId>
             <artifactId>jta</artifactId>
             <version>${jta.api.version}</version>
-            <scope>provided</scope>
```

Comment:
Sorry I don't follow. It's not provided anymore so do you suggest to introduce it with scope `compile` and as optional dep right?